OpenRA.Game/Effects/FlashTarget.cs

```diff
@@ -25,7 +25,7 @@ public FlashTarget(Actor target, Player asPlayer = null, int ticks = 4)
 			this.target = target;
 			player = asPlayer;
 			remainingTicks = ticks;
-			foreach (var e in target.World.Effects.OfType<FlashTarget>().Where(a => a.target == target).ToArray())
+			foreach (var e in target.World.Effects.OfType<FlashTarget>().Where(a => a.target == target).ToList())
```

Comment:
You need to take a copy (using either ToArray or ToList) and enumerate over that instead if you want to remove items while you are enumerating over it with foreach. This avoids a sanity-check exception which would otherwise be thrown.